src/python/pants/jvm/test/junit.default.lockfile.txt

```diff
@@ -130,9 +130,9 @@ version = "1.7.2"
 packaging = "jar"
 
 [[entries.dependencies]]
-group = "org.junit.platform"
-artifact = "junit-platform-commons"
-version = "1.7.2"
+group = "org.apiguardian"
```

Comment:
Hrm, I missed this in the main review. There is no --python-only flag to the re-gen script; so I had just reverted all JVM lockfile updates when I did the initial upgrade on main to 2.1.95. It's probably fine though / tests will cover the lock update as they always do or don't.